executor/write.go

```diff
@@ -1855,9 +1852,9 @@ func (e *ReplaceExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 
 	var rows [][]types.Datum
 	if len(e.children) > 0 && e.children[0] != nil {
-		rows, err = e.getRowsSelectChunk(ctx, cols, false)
+		rows, err = e.getRowsSelectChunk(ctx, cols)
```

Comment:
This line is changed

Reply:
This line and the below one are in `replace` statement, due to there is no `replace ignore` syntax, so they are always be `false`.